nextflow.config

```diff
@@ -71,14 +72,14 @@ params {
     krakenuniq_keepintermediate  = false
     build_malt                   = false
 
-    bracken_build_options         = ''
-    centrifuge_build_options      = ''
-    diamond_build_options         = ''
-    ganon_build_options           = ''
-    kaiju_build_options           = ''
-    kraken2_build_options         = ''
-    krakenuniq_build_options      = ''
-    malt_build_options            = "--sequenceType DNA"
+    bracken_build_options        = ''
+    centrifuge_build_options     = ''
+    diamond_build_options        = ''
+    ganon_build_options          = ''
+    kaiju_build_options          = ''
+    kraken2_build_options        = ''
+    krakenuniq_build_options     = ''
+    malt_build_options           = "--sequenceType DNA"
```

Comment:
Shouldn't this be up to the user and therefore `' '`?

Reply:
The vast majority of cases people will use DNA so we set this as default

Reply:
OK, resolving this.